tests/integration/assessment/test_assessment.py

```diff
@@ -165,3 +166,18 @@ def test_spn_crawler_with_available_secrets(
     ws.secrets.delete_secret(scope=secret_scope, key=secret_key)
 
     assert len(results) >= 2
+
+
+def test_workspace_object_crawler(ws, make_notebook, make_directory, inventory_schema, sql_backend):
+    new_notebook = make_notebook()
+    new_directory = make_directory()
+    workspace_listing = WorkspaceListing(ws=ws, sql_backend=sql_backend, inventory_database=inventory_schema)
```

Comment:
Add start path, otherwise this test will scan the entire workspace

Reply:
done